README.md

```diff
@@ -101,6 +101,8 @@ Sources for the integration test are kept in `src/integrationTest`.
 Building a release requires [install4j]. We've got an [Open Source License](https://www.ej-technologies.com/buy/install4j/openSource).
 Releasing is done using [CircleCI](https://circleci.com/gh/JabRef/jabref). A full release howto is available [in our wiki](https://github.com/JabRef/jabref/wiki/Releasing-a-new-Version).
 
+## Rights of Loading Animation
+The loading animation during loading of recommendations from Mr. DLib is created by http://loading.io/ and is free of use.
```

Comment:
Please provide a concrete license. Is it CC-0? https://creativecommons.org/publicdomain/zero/1.0/